app/Console/Commands/Tests/SetupFrontEndTestUser.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Console\Commands\Tests;
+
+use App\Models\User\User;
+use Illuminate\Console\Command;
+
+class SetupFrontEndTestUser extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'setup:frontendtestuser
+                            {--database= : The database connection to use}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a user test exclusively for front-end testing with Cypress.';
+
+    /**
+     * The migrator instance.
+     *
+     * @var \Illuminate\Database\Migrations\Migrator
+     */
+    protected $migrator;
+
+    /**
+     * Create a new command.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->migrator = app('migrator');
```

Comment:
I don’t understand this.

Reply:
This allow to use the command on another database with the `--database` option.